index.html

```diff
@@ -119,11 +146,14 @@ <h1>Profesional Art Printing Data</h1>
           <a href="#"><li class="project-item-3">Ruby</li></a>
         </ul>
       </div>
+   </div>
       <div class="project-list">
         <a href="#">See Project</a>
       </div>
+       
       </div>
-      <div class="project">
+      <div class="four-project">
+        <div class="project-content">

```

Comment:
- [ ] Also consider making project cards like the one in the template in height and width and also fix height for it in the mobile version and also make sure overlay content.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/200132814-e7a241b1-b13c-4c82-890a-36561ae5785a.png)  | ![image](https://user-images.githubusercontent.com/53402600/200132795-ee48d786-d52e-43f7-9ffe-26a86bd8270b.png) |